vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,53 +1,91 @@
+You can create a hierarchy at any level.
 
-Following this recipes example, you can try creating other notes that might be relevant. Some examples: `recipes.italian.appetizers`, `recipes.chinese`, `recipes.chinese.appetizers`, `recipes.ingredients.favorites`, `recipes.ingredients.shopping-list`. Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to easily change the hierarchies later if you change your mind on how to organize your notes.
+> 🌱 **ACTION:** Type the following into lookup and hit enter: `recipes.italian.desserts.tiramisu`
+
+You'll there is a `+` sign next to `italian` and `desserts` when viewing in the lookup or Tree View. The plus sign indicates that this note is a stub. A **[[stub|dendron://dendron.dendron-site/dendron.concepts#stubs]]** is a placeholder for a note that hasn't been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
+
+![[dendron://dendron.dendron-site/asset.preview#tree-view-dark,1:#*]]
+
+Following this recipes example, you can create other related. Some examples:
+
+```
+recipes.favorites
+recipes.italian.appetizers
+recipes.chinese
+recipes.chinese.appetizers.wontons
+recipes.ingredients.favorites
+recipes.ingredients.shopping-list
+```
+
+You may be wondering why Dendron uses hierarchies. You can [[read here|dendron://dendron.dendron-site/dendron.faq#why-hierarchies]] for our thoughts about structure.
 
 ### Finding Notes
 
 To find notes, we use the same lookup interface that we used to create them.
 
-> 🌱 Open Lookup, and type `vege`. This will find your `recipes.vegetarian` note. Hit `Enter` to open that note.
+> 🌱 **ACTION:** Open Lookup, and type `vege`. This will find your `recipes.vegetarian` note. Hit `Enter` to open that note.
+
+Lookup uses _fuzzy search_, which means you can type out partial queries and still see the results. Entering multiple keywords delimited by space will lookup matching notes regardless of order of the keywords.
 
-The lookup uses fuzzy search which means you can type out partial results and still see the results. Searching with * wildcards is supported. Also entering multiple keywords delimited by space will look up matching notes regardless of order of the keywords. Example: `vege rec`/`rec vege` will both match `recipes.vegetarian` note. Further info on looking up notes can be found in [[dendron.topic.lookup.find]]
+- Example: `vege rec` or `rec vege` will match the `recipes.vegetarian` note.
 
-When combined with hierarchies, this fast lookup system enables you to find your notes very quickly, even if you have thousands of notes in your vaults.
+Further info on using lookup to find notes can be found in [[dendron://dendron.dendron-site/dendron.topic.lookup.find]].
 
-If you want to search for content within the notes, you can use VS Code's built-in search tools with `%KEYBINDING%+Shift+F`.
+![[dendron://dendron.dendron-site/dendron.topic.search]]
```

Comment:
this feels like an aside since we were just talking about lookup. maybe put this in a >TIP section or under a separate header